primitiv/opencl_device.cc

```diff
@@ -739,7 +216,14 @@ struct OpenCLInternalState {
           // Then, we can delete the buffer safely.
           delete static_cast<cl::Buffer *>(ptr);
         }) {
-      cl::Program program(context, ::generate_kernels(), true);
+      cl::Program program(context, ::generate_kernels());
+      try {
+        program.build({device});
+      } catch (...) {
+        std::cerr << program.getBuildInfo<CL_PROGRAM_BUILD_LOG>(device)
```

Comment:
Should not directly print any messages to the standard streams without users' request.
Using `THROW_ERROR` instead to inform this message.